adapters/InMobi/InMobiAdapter/GADMAdapterInMobiRewardedAd.m

```diff
@@ -138,6 +138,11 @@ - (void)requestRewardedAd {
     [_rewardedAd setKeywords:extras.keywords];
   }
 
+  if (_adConfig.watermark != nil) {
+    IMWatermark *watermark = [[IMWatermark alloc] initWithImageData:_adConfig.watermark];
+    [_rewardedAd setWatermarkWith:watermark];
+  }
+    
```

Comment:
Nit: Our internal code review system detects empty spaces in this line, can you remove them?